examples/auctions/simple_open_auction.v.py

```diff
@@ -0,0 +1,76 @@
+# Open Auction
+
+# Auction params
+# Beneficiary recieves money from the highest bidder
+beneficiary: public(address)
+auction_start: public(timestamp)
+bidding_time: public(num(sec))
+
+# Current state of auction
+highest_bidder: public(address)
+highest_bid: public(num(wei))
+
+# Allowed withdrawals of previous bids
+pending_returns:(num(wei)[address])
+
+# Set to true at the end, disallows any change
+ended: public(bool)
+
+# Create a simple auction with `_bidding_time`
+# seconds bidding time on behalf of the
+# beneficiary address `_beneficiary`.
+def __init__(_beneficiary: address, _bidding_time: num(sec)):
+    self.beneficiary = _beneficiary
+    self.bidding_time = _bidding_time
+    self.auction_start = block.timestamp
+
```

Comment:
`self.ended` does default to `False`  but I agree that setting it to `False` in `__init__` is clearer and hence should be done.